src/Stratis.Bitcoin.Features.Api/ApiFeature.cs

```diff
@@ -117,29 +119,42 @@ public override void Dispose()
         }
     }
 
-    public sealed class ApiFeatureOptions
+    public sealed class ApiSettingsDefaults
```

Comment:
This looks good, I only question the naming as it seems to be dual-purpose. There are some defaults but they can be overridden by the user.

Reply:
Nevermind, I missed the place where this is consumed and see that it is being used as defaults. In that case, should these properties be immutable?

Reply:
See below.